README.md

```diff
@@ -205,7 +211,26 @@ we decided to adhere to in this project.
 * Markdown files (markdownlint standard rules)
 
   ```sh
-  docker compose run --rm node yarn coding-standards-check/markdownlint
+  docker run --rm --interactive --tty --volume ${PWD}:/app node:18 yarn --cwd /app install
+  docker run --rm --interactive --tty --volume ${PWD}:/app node:18 yarn --cwd /app coding-standards-check
+  ```
+
+### Coding standards apply
+
+The following commands let you apply the coding standards
+we decided to adhere to in this project.
+
+* PHP files (PHP-CS-Fixer with the Symfony ruleset enabled)
+
+   ```sh
+   docker compose exec phpfpm composer coding-standards-apply
+   ```
+
+* Markdown files (markdownlint standard rules)
+
+  ```sh
```

Comment:
Inconsistent indentation (2 and 3 spaces before backticks).

Reply:
Fixed.